themes/vex-hugo/layouts/partials/banner.html

```diff
@@ -4,7 +4,7 @@
 {{- with $data.homepage.banner }}
 <!-- hero area -->
 <section class="d-lg-none d-flex align-items-center">
-	<img src="{{ .image | absURL }}" class="w-100" style="height: 300px;" alt="">
+	<img src="{{ .image | absURL }}" class="w-100" width="454" height="300" alt="">
```

Comment:
An alt text (e.g. as parameter) would be great here.

Reply:
done